### PR TITLE
JAVA-11533 Move core-java related modules to core-java-modules

### PR DESCRIPTION
**Pull request by anuragkumawat**

### CODE REVIEW

File: core-java-modules/pom.xml [hunk 1]
```diff
@@ -118,6 +118,9 @@
         <module>core-java-regex-2</module>
         <module>core-java-uuid</module>
         <module>pre-jpms</module>
+        <module>java-collections-conversions</module>
```
Comment (dkapil) [inline core-java-modules/pom.xml:121]: Can you insert these new modules alphabetically in this list?

**Review by dkapil — CHANGES_REQUESTED** [review]

File: core-java-modules/pom.xml (hunk 1 above)
Comment (anuragkumawat) [reply core-java-modules/pom.xml:120]: Addressed

**Review by dkapil — APPROVED** [review]

**Comment by dkapil** [comment]
test this please

**Comment by anuragkumawat** [comment]
@dkapil  build has passed successfully.

**Pull request merged**